prow/plugins.yaml

```diff
@@ -287,6 +287,15 @@ require_matching_label:
   prs: true
   regexp: ^(sig|wg|committee)/
   # Define needs-sig separately for issues so that we only comment on issues.
+- missing_label: needs-sig-triage
```

Comment:
What are you trying to accomplish? Right now it looks like this will add a `needs-sig-triage` until someone adds a `needs-sig` label??

Reply:
@fejta have a quick read through https://github.com/kubernetes/community/issues/3456 and the last comments - the `needs-triage` label will be there to make it easier for SIGs to sort through new incoming issues.

the 'exit criteria' of `needs-triage` will be a `sig`, `lifecycle` and maybe `priority` labels applied, first we need some consensus to get this fixed and merged.

Reply:
I've changed it to simply await `lifecycle`, as `kind` is covered by `needs-kind`. 

Lifecycle will be intented to show the status of the issue - 'active' as in-progress, 'ready' as in 'ready to be worked at', 'frozen' as in backlog/longterm

Reply:
@fejta -- I think what I'm imagining is that `needs-triage|needs/triage` label gets removed from the issue/PR once the following are satisfied:
- has `sig/*`
- has `kind/*`
- has `priority/*`

Is there an easy way to represent that in this plugin?

I don't think a `lifecycle/ready` label is necessary, as it's the same `-label:needs-triage|needs/triage`.

Reply:
From my overall experience with triage I think that forcing all three labels is kinda too much. There are always tickets that don't need some of them, especially priority. In an ideal world it would be good, but it's too many labels to apply, on an OSS basis. Also there are already mechanisms for them (needs-kind etc).

`-label:needs-triage` will show stale, rotten, frozen and many others. lifecycle/ready can be triaged issues that are ready to be picked for work , i.e. "prioritized"